management-web/src/main/java/com/ns/task/product/ProductController.java

```diff
@@ -0,0 +1,47 @@
+package com.ns.task.product;
+
+import com.mx.task.model.ProductDTO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import javax.validation.Valid;
+
+@Controller
+public class ProductController {
+
+
+    ProductService productService;
+
+    @Autowired
+    public ProductController(ProductService productService) {
+        this.productService = productService;
+    }
+
+    @GetMapping("product")
```

Comment:
is not necessary to have the `/` at the beginning of the URL?

Reply:
I check and is not difference if I write the preceding slash or not, but I add it be consistent